reference/carts.v3.yml

```diff
@@ -1728,13 +1729,11 @@ components:
                         valueId:
                           type: number
                           description: The product option value identifier.
```

Comment:
I would consider adding, "in number format." and possibly provide an example of a value id